sematic/api/endpoints/artifacts.py

```diff
@@ -30,15 +29,17 @@
 @sematic_api.route("/api/v1/artifacts", methods=["GET"])
 @authenticate
 def list_artifacts_endpoint(user: Optional[User] = None) -> flask.Response:
-    limit, _, _, sql_predicates = get_request_parameters(flask.request.args, Artifact)
+    limit, order, _, _, sql_predicates = get_request_parameters(
```

Comment:
Thanks @tscurtu :) 

Later, we should also decide how to deal with the `modal` parameter of `get_request_parameters`. Since each view function requires different models. It is a little hard to do `_get_sql_predicates` universally. Some options I can think of are:

1.  Keep using a function like `get_request_parameters` from each view function to do the unique logic; the standard parameter processing logic is taken out and placed in [url_value_preprocessor](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Flask.url_value_preprocessor)

2. Add a decorator for each view function that needs `_get_sql_predicates`, and attach the `model` type. The decorator wraps the view function and puts the `view function` -> `model` into a map. Then, in url_value_preprocessor, it retrieves the view function by doing [this](https://stackoverflow.com/questions/38488134/get-the-flask-view-function-that-matches-a-url). Then read the `model` type from the map.

3. Use a decorator, but do not use the map. Same as option 2, after getting the view function, find out what decorator is used to determine the model type. [Here](https://stackoverflow.com/questions/3232024/introspection-to-get-decorator-names-on-a-method) mentions several ways to do that. 

The last two might feel over-engineering, though.